internal-tooling/performanceUtils.ts

```diff
@@ -17,16 +17,37 @@
 import {execSync} from 'child_process';
 import {mkdirSync, mkdtempSync, unlinkSync} from 'fs';
 import * as path from 'path';
+import yargs = require('yargs');
 import {Bucket, Storage, TransferManager} from '../src';
 
 export const BLOCK_SIZE_IN_BYTES = 1024;
-export const DEFAULT_SMALL_FILE_SIZE_BYTES = 5120;
-export const DEFAULT_LARGE_FILE_SIZE_BYTES = 2.147e9;
 export const NODE_DEFAULT_HIGHWATER_MARK_BYTES = 16384;
 export const DEFAULT_DIRECTORY_PROBABILITY = 0.1;
-export const DEFAULT_PROJECT_ID = 'GCS_NODE_PERFORMANCE_METRICS';
 export const DEFAULT_NUMBER_OF_OBJECTS = 1000;
+export const enum OUTPUT_FORMATS {
```

Comment:
I see these already existed, but maybe we want an [`as const`](https://www.typescriptlang.org/docs/handbook/enums.html#objects-vs-enums) object rather than [`const enum`](https://www.typescriptlang.org/docs/handbook/enums.html#const-enums).

Reply:
Done